src/Resources/views/Invoice/Download/pdf.html.twig

```diff
@@ -1,125 +1,109 @@
 {% set shopBillingData = invoice.shopBillingData %}
 
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-    </head>
-    <body>
-        <h1>{{ 'sylius_invoicing_plugin.ui.invoice'|trans }} {{ invoice.number }}</h1>
-        <p>{{ 'sylius_invoicing_plugin.ui.order_number'|trans }}: {{ invoice.orderNumber }}</p>
-        <p>{{ 'sylius_invoicing_plugin.ui.issued_at'|trans }}: {{ invoice.issuedAt|date('Y-m-d H:i:s') }}</p>
+<head>
+    <meta charset="utf-8">
+    <style>
+        .invoice-box { max-width: 800px; margin: auto; padding: 30px; border: 1px solid #eee; box-shadow: 0 0 10px rgba(0, 0, 0, .15); font-size: 16px; line-height: 24px; font-family: 'Helvetica Neue', 'Helvetica', Helvetica, Arial, sans-serif; color: #555 }
+        .invoice-box table { width: 100%; line-height: inherit; text-align: left; }
+        .invoice-box table td { padding: 5px; vertical-align: top; text-align: right; }
+        .invoice-box table tr td:nth-child(1) { text-align: left; }
+        .invoice-box table tr.top table td { padding-bottom: 20px; }
+        .invoice-box table tr.top table td.title { font-size: 45px; line-height: 45px; color: #333; }
+        .invoice-box table tr.information table td { padding-bottom: 40px; }
+        .invoice-box table tr.heading td { background: #eee; border-bottom: 1px solid #ddd; font-weight: bold; }
+        .invoice-box table tr.details td { padding-bottom: 20px; }
+        .invoice-box table tr.item td{ border-bottom: 1px solid #eee; }
+        .invoice-box table tr.item.last td { border-bottom: none; }
+        .invoice-box table tr.total td { border-top: 2px solid #eee; font-weight: bold; }
+    </style>
+</head>
 
-        <br /><br />
+<body>
+<div class="invoice-box">
+    <table cellpadding="0" cellspacing="0">
+        <tr class="top">
+            <td colspan="4">
+                <table>
+                    <tr>
+                        <td colspan="3" class="title">
+                            <img src="https://sylius.com/assets/sylius-logo.png" style="width:300px;">
```

Comment:
WDYT about downloading the picture and including it into the project? I'm not sure if we should rely on an external resource in such case like generating an invoice.